service/src/main/resources/db/changesets/20230410_schema_reset.yaml

```diff
@@ -408,22 +409,22 @@ databaseChangeLog:
             columns:
               - column:
                   name: id
-                  type: ${id.type}
+                  type: text
                   constraints:
                     nullable: false
               - column:
                   name: criteria_group_id
-                  type: ${id.type}
+                  type: text
                   constraints:
                     nullable: true
               - column:
                   name: criteria_group_section_id
-                  type: ${id.type}
+                  type: text
                   constraints:
                     nullable: true
               - column:
                   name: cohort_revision_id
-                  type: ${id.type}
+                  type: text
                   constraints:
                     references: cohort_revision(id)
                     foreignKeyName: ck_crit_cr
```

Comment:
Foreign key name: **ck_crit_cr** `20230410_schema_reset.yaml#430`  clashes with same name in `20230530_avoid_postgres_specific_features.yaml#65` Please consider renaming one of these to a different name maybe `ck_crit_cr_2` in the `avoid_postgres` file